crates/bitwarden/src/auth/client_auth.rs

```diff
@@ -90,6 +92,10 @@ impl<'a> ClientAuth<'a> {
     pub async fn send_two_factor_email(&mut self, tf: &TwoFactorEmailRequest) -> Result<()> {
         send_two_factor_email(self.client, tf).await
     }
+
+    pub async fn validate_password(&self, password: String, user_key: String) -> Result<bool> {
+        validate_password(self.client, password, user_key).await
```

Comment:
If we are checking the password against the password hash, `user_key` should be renamed or it's going to cause confusion